jobs/index.mjs

```diff
@@ -19,8 +19,8 @@ export default [
   },
   {
     name: "vaults",
-    interval: "6m",
-    timeout: 20000,
+    interval: "7m",
+    timeout: 40000,
```

Comment:
not sure why I had to increase the delay for when the vaults job first starts for it to pass, I think adding more work with the arbitrium addition meant it couldn't complete it simultaneously as the other jobs still.. might just be my machine not being beefy enough and it would be fine in production, not sure..